int-tests/test_server.py

```diff
@@ -1,10 +1,19 @@
 import unittest
-import requests
+from unsafe_client import http_get
 import os.path as path
 from server_controller import Server, Initializer, module_dir
+from unittest import TestCase, skip
 
 
-class TestReverseProxyServer(unittest.TestCase):
+class TestReverseProxyServer(TestCase):
+
+  @classmethod
+  def setUpClass(cls):
+    global back_init
+    back_init = Initializer('int-test-backend-server')
+    global front_init
+    front_init = Initializer('int-test-proxy-server')
```

Comment:
even though Gradle has incremental builds - there's no need to invoke the artifact build process more than once